server/middleware/fetchFrontendComponentMiddleware.ts

```diff
@@ -1,21 +1,9 @@
 import { RequestHandler } from 'express'
-import type { FrontendComponentsPageAdditions } from 'viewModels'
-import logger from '../../logger'
 import { Services } from '../services'
 
 export default function getFrontendComponents({ frontendComponentService }: Services): RequestHandler {
   return async (req, res, next) => {
-    try {
-      const footer = await frontendComponentService.getComponents('footer', res.locals.user.token)
-      res.locals.feComponents = {
-        footerHtml: footer.html,
-        cssIncludes: [...footer.css],
-        jsIncludes: [...footer.javascript],
-      } as FrontendComponentsPageAdditions
-      next()
-    } catch (error) {
-      logger.error(error, 'Failed to retrieve front end components')
-      next()
-    }
+    res.locals.feComponents = await frontendComponentService.getComponents(res.locals.user.token)
+    next()
```

Comment:
The middleware is now much much simpler 😁 as it does not need to worry about error handling (as the service does that) and it does not need to worry about mapping (again, the service does that)